test/test_automated_install.py

```diff
@@ -310,15 +310,16 @@ def test_FTL_download_unknown_fails_no_errors(Pihole):
     error = 'Error: URL not found'
     assert error in download_binary.stdout
 
-def test_FTL_binary_installed_and_responsive_no_errors(Pihole):
-    ''' confirms FTL binary is copied and functional in installed location '''
-    installed_binary = Pihole.run('''
-    source /opt/pihole/basic-install.sh
-    FTLdetect
-    pihole-FTL version
-    ''')
-    expected_stdout = 'v'
-    assert expected_stdout in installed_binary.stdout
+# Temporarily disabled as we cannot use setcap on Travis CI
```

Comment:
Will this be re-enabled when we use CircleCI then?

Reply:
Yes

Reply:
If it's running in the docker container, woudn't it be fine to run on Travis?

Reply:
Travis doesn't like us playing with `setcap`